### DISCUSSION TITLE
Question - Possible to Stop and Start Replay?

### DISCUSSION THREAD

**Discussion by AlephNullNull** [ruffle-rs/ruffle#4283] [Q&A] [answered]
Hi Folks!

I have some 10 year old educational videos, some in SWF format, that I am trying to convert.  I've tried Handbrake, Ffmpeg, VLC, etc. without any luck.  Ruffle plays the videos well and I am down to screen recording the videos to save them in a more modern format.  However, I have not found a way to start and stop playback with Ruffle after loading the SWF.  Is there a way to do that?

Also, if anyone had any ideas on other tools that I could potentially explore to do SWF conversion, any pointers appreciated!

Thank you!

Cheers!

**Comment by Bale001** [comment, marked as answer]
The latest version of Ruffle lets you control playback in the right click menu. If that doesn't work, have you tried [Swivel](https://github.com/Herschel/Swivel)?

**Reply by AlephNullNull** [reply]
Thanks, @EmperorBale 

I did not know about either the  right-click or Swivel..  Thanks much!  It looks like Swivel does not work on Mac OS Big Sur but I have access to a Windows machine so I will try that.

Much appreciate your help.  Thanks, again!

**Reply by ousia** [reply]
@AlephNullNull, did @EmperorBale’s reply answer your question?

In that case, it would be great that you mark it as answered.
